bench-show.cabal

```diff
@@ -60,10 +60,7 @@ description:
 category:            Performance, Benchmarking
 homepage:            https://github.com/composewell/bench-show
 license-file:        LICENSE
-tested-with:           GHC==8.6.5
-                     , GHC==8.4.4
-                     , GHC==8.2.2
-                     , GHC==7.10.3
+tested-with:         GHC==8.8.3
```

Comment:
Since nothing has changed in the source, it should still be compiling with all other compilers. It won't hurt to add 8.6.5/8.4.4 to the build matrix and still have those here.